include/config.h

```diff
@@ -52,7 +52,14 @@
 #define SCREEN_HEIGHT           240         // Default height is 240. Recommended to be 224 for 4MB users.
 
 
+/* --------------------------Usb-------------------------- */
```

Comment:
This wants to be further below, with the rest of the safety checks underneath `// Don't touch any of this below.`